installer/config.json.example

```diff
@@ -39,5 +39,7 @@
     "TxSyncIntervalSeconds": 60,
     "TxSyncServeResponseSize": 1000000,
     "TxSyncTimeoutSeconds": 30,
-    "ForceRelayMessages": false
+    "ForceRelayMessages": false,
```

Comment:
Can you please re-alphabetize this file -- I would like to keep it alphabetized for easy reference.

Reply:
yes